### ISSUE TITLE
Technical support request

### ISSUE THREAD

**Issue by GerBreOwn** [inlets/inlets-operator#71]
## Expected Behaviour
 I should NOT get the above error

## Steps to Reproduce (for bugs)

1.I enter "inlets client --remote=ws://xxx.xxx.xxx.xxx --token XXXXX... --upstream="with new domain=http://192.168.1.141"
2.my inlets is version 2.7.0
3.This was working in the past.
4.

## Context

I am trying to connect a new domain on my Raspberry Pi to my DO droplet. NO DOCKER, NO KUBERNETES.

## Your Environment
Raspian Buster on Raspberry Pi 2. 2 domains on 2 Raspberry 3 B+ and 1 domain on Raspberry Pi 4 all of which are running OK now but when I try to add another domain and run "inlets client" I have started getting this error. I have also reported this to DO but so far no response from them.

* inlets-operator version, find via `kubectl get deploy inlets-operator -o wide`
NONE

* Kubernetes distribution i.e. minikube v0.29.0., KinD v0.5.1, Docker Desktop:
NONE

* Kubernetes version `kubectl version`:
NONE

* Operating System and version (e.g. Linux, Windows, MacOS):
Raspian Buster Linux. One domain is powered by Joomla, 1 by wordpress and 1 by Gitea.

* Cloud provisioner: (DigitalOcean.com / Packet.com / etc)
DigitalOcean.com

**Comment by alexellis** [comment]
Hi, thank you for your interest in the project.

Unfortunately, the information you have provided is insufficient to be able to help you. Please share the exact command you are using at both ends of the tunnel. From the error, it looks like you may have misconfigured a proxy that you are using and haven't told us about, or perhaps you have the token wrong at one end? 

Alex

**Comment by GerBreOwn** [comment]
Initially I entered inletsctl create \
 --provider digitalocean \
 --access-token-file ~/do-token \
 --region sfo1
Which created a DO droplet.
On my client system I enter "inlets client --remote=ws://xxx.xxx.xxx.xxx --token XXXXX... --upstream="with new domain=http://192.168.1.141:80"
I have NOT configured any proxy that I know of.
I generated a new token at DO and entered it in the above inlets client statement.
I reread the your blog "raspberry-pi-zero-tunnel-g...". I checked the "/etc/default/inlets" file and saw the token was an old one so I changed that to the new one I received last night and also added the new domain to the upstream. I then did a daimon-reload and inlets restart but I am still getting the same error: "Failed to connect to proxy. Response status: 200 - 200 OK. Response body: failed authentication  error="websocket: bad handshake". I was trying to add a new domain to my upstream by running the "inlets client" command  when I started getting  the error. Previously it was working fine as I am able to run 2 blog sites on 2 of my Raspberry Pi computers..
What else can I do at my end to make the error go away or is it something that needs to be done at DO's end?
Thanks for your help.
